indico/web/client/js/jquery/utils/forms.js

```diff
@@ -258,6 +260,13 @@
       lockedForms.find(':input:not([type=hidden]):not([data-button-back])').prop('disabled', true);
       lockedForms.find(':input:submit').hide();
     }
+
+    if (forms.data('confirm-close-unsaved') !== undefined) {
```

Comment:
this is something you need to check for each for individually; `forms` can contain more than one